src/cmd/property.go

```diff
@@ -15,18 +16,63 @@ var examplePropertyCmd = &cobra.Command{
 	Use:     "property",
 	Aliases: []string{"prop"},
 	Short:   "Example Property",
-	Long:    `Example Property`,
+	Long:    "Example Property",
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println(getExample[opslevel.PropertyInput]())
 	},
 }
 
+var examplePropertyDefinitionInputFileYAML = `
+name: Name and age property using YAML
+description: Tracks name and age
+propertyDisplayStatus: visible
+allowedInConfigFiles: true
+schema:
+  type: object
+  required:
+    - name
+  properties:
+    name:
+      type: string
+    age:
+      type: number
+`
+
+var examplePropertyDefinitionInputFileJSON = `
+name: Name and age property using JSON
+description: Tracks name and age
+propertyDisplayStatus: hidden
+allowedInConfigFiles: false
+schema: |
+  {
+      "type": "object",
+      "required": [
+          "name"
+      ],
+      "properties": {
+          "name": {
+              "type": "string"
+          },
+          "age": {
+              "type": "number"
+          }
+      }
+  }
+`
+
+func propertyDefinitionExample() string {
+	if !isYamlOutput() {
```

Comment:
this var is populated by the flag `-o` and is only to be used when the end user does `-o yaml` on a `get` or `list` command.  It looks like you are using this function in the help text for a command.

This is conflating functionality and is not what `isYamlOutput` was intended for.  I'm not sure why we would show the `examplePropertyDefinitionInputFileJSON` in help text but you should beable to just put both examples into the help text if there is a reason for the JSON version.

IMO i would much rather we only support the `examplePropertyDefinitionInputFileYAML` and we throw an error when someone does the multiline string JSON version.